module-types/Datacom/PSU-400-AC-B.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 400 AC-B
+part_number: 820.0011.xx
+comments: AC power supply with 100-240VAC and 50/60Hz input. Forced ventilation with airflow entering through the front panel of the power supply. Allows
+  operation as a redundant and hot-swap power source.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be